### ISSUE TITLE
Add endpoint for querying roles?

### ISSUE THREAD

**Issue by alechenninger** [lightblue-platform/lightblue-rest#253]
I'd like an endpoint I could call from my application to confirm that my application can talk to lightblue, can authenticate, and has expected roles. This would be used as a healthcheck.

Good idea? Bad idea? Feasible?

**Comment by Abhikos** [comment]
Good idea, Also if we can check what access permission a client certificate is granted by hitting an end point?

**Comment by derek63** [comment]
+1 to endpoint with roles
